.github/workflows/test-conda.yml

```diff
@@ -0,0 +1,55 @@
+name: Test using Conda environments
+
+on: [push, pull_request]
+
+jobs:
+  TestConda:
+    name: ${{ matrix.os }}, ${{ matrix.env }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        env:
+          - ci/envs/py36-geos36.yml
+          - ci/envs/py37-geos37.yml
+          - ci/envs/py38-geos38.yml
+          - ci/envs/py38-geos-latest.yml
+          - ci/envs/py38-geos-latest-conda-forge.yml
```

Comment:
For such a small env, I have the feeling that the environment.yaml files are a bit overkill, and having here matrix variables for python/numpy/geos version is more flexible to have certain combinations.